README.md

```diff
@@ -1,2 +1,20 @@
-# TAC
-Automating Compliance Tooling Project - Technical Advisory Council
+# [Automating Compliance Tooling (ACT)](https://automatecompliance.org/)
+Our mission is to support development of open source tooling for efficient and effective exchange of software bill of materials (SBOM) to enable license compliance, security, export control, pedigree and provenance workflows.
+
+## Current Projects under the ACT umbrella
+* [FOSSology](https://www.fossology.org/)
+* [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
+* [SPDX Tools](https://github.com/spdx/tools)
+* [Tern](https://github.com/tern-tools/tern)
+* [QMSTR](https://qmstr.org/)
+
+## ACT Affiliate Projects
```

Comment:
Affiliate projects are ACT projects not under Linux Foundation project governance or charters.